siatune/core/rewriters/path.py

```diff
@@ -1,12 +1,25 @@
 # Copyright (c) SI-Analytics. All rights reserved.
+import os
 from os import path as osp
 
 from ray.air import session
 
+from siatune.utils import ref_raw_args
 from .base import BaseRewriter
 from .builder import REWRITERS
 
 
+@REWRITERS.register_module()
+class RevertWorkSpace(BaseRewriter):
+
+    def __init__(self, key='tune_launch_path'):
+        self.key = key
+
+    def __call__(self, context: dict) -> dict:
+        os.chdir(context.pop(self.key))
```

Comment:
If you're not using the `DataParallelTrainer` from ray.train, you don't need to use `os.chdir`. Instead, you can set `chdir_to_trial_dir` to `True` in your `tune_config`.